services/121-service/src/user/user.service.ts

```diff
@@ -183,7 +172,7 @@ export class UserService {
       {
         id: user.id,
```

Comment:
I think we can leave the (internal) user-id out of the JWT.

Reply:
done in next commit